Week-2/06 CSS Advanced/Recipe with flexbox/oatmeal_cookies.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>binary eats - Oatmeal Coconut Pantry Cookies</title>
+      <link href="styles.css" rel="stylesheet">
+   </head>
+
+   <body>
+      <header>
+         <div class="logo">binary eats</div>
+         <div>
+            <a href="">sign-in</a>
+            <a href="">search</a>
+         </div>
+      </header>
+      <div class="content">
+         <div class="main-content">
+            <div class="title">
+               <h1>OATMEAL COCONUT PANTRY COOKIES</h1>
+               <p>Chef Roman, August 24, 2018</p>
+            </div>
+            <div class="summary">
+               <img src="oatmealCookie_small.jpg" alt="Photo of two of the world's best oatmeal cookies">
```

Comment:
try to use relative path, and change it everywhere you are using it 
even if that is in the same directory